apps/comments-ui/test/e2e/options.test.ts

```diff
@@ -193,7 +193,15 @@ test.describe('Options', async () => {
 
         test('Uses 100 avatarSaturation', async ({page}) => {
             const mockedApi = new MockedApi({});
-            mockedApi.addComment();
+            mockedApi.addComment({
+                member: {
+                    id: 'test-id',
+                    uuid: 'test-uuid',
+                    name: 'Test User',
+                    avatar: '',
+                    expertise: ''
+                }
+            });
```

Comment:
Why was this needed?

Reply:
This test was failing, and I didn't understand why – I hadn't touched the avatarSaturation, the only thing that changed is that the Avatar component isn't passed in the Form anymore but in FormWrapper. Cursor suggested this because in FormWrapper, I'm passing member directly as a prop to Avatar, while the test is expecting to find an Avatar that's rendered for a comment.